### DISCUSSION TITLE
Add the option for actual weather animation instead of geometric

### DISCUSSION THREAD

**Discussion by ToughDBlue** [breezy-weather/breezy-weather#1162] [Ideas]
### Describe your idea

Please add the ability to choose between the current geometric weather animation and an actual weather animation.

### Other details

I know it's not FOSS but to get an idea of what I'm suggesting please see the stock weather app in iOS which I think is Dark Sky.

### Acknowledgements

- [X] I have searched the [existing issues](https://github.com/breezy-weather/breezy-weather/issues) and [existing discussions](https://github.com/breezy-weather/breezy-weather/discussions/categories/ideas) and this is a new ticket, **NOT** a duplicate or related to another open OR closed issue or discussion.
- [X] The idea I’m suggesting is not in the [list of features that will not be implemented](https://github.com/breezy-weather/breezy-weather/blob/main/README.md#features-that-will-not-be-implemented).
- [X] I have updated the app to **[version 5.2.5 or later](https://github.com/breezy-weather/breezy-weather/releases/latest)**. Even if I’m getting updates from F-Droid default repo, I don’t need to uninstall/reinstall to update.
- [X] I wrote my request in English, and any screenshot I attached are also in English (language can be changed from Settings > Appearance). I can write in my native language under the English text, if necessary.
- [X] I understand that this app is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY, so I may not get a positive reply to my request, if any at all.

**Comment by papjul** [comment]
Sorry, I don't have iOS so I have no idea what this option is? By default, the background animation is already the actual weather, and you can choose to force a specific weather condition.

**Comment by ToughDBlue** [comment]
Sorry I didn't clarify what I meant. I'm not asking for the actual weather but that the animation is a real actual video picture of what the weather is and not a geometric animation.
![tmp_e585a86d-3856-48d0-be7a-71cfc331345e_1](https://github.com/breezy-weather/breezy-weather/assets/53195842/bd04db42-c8b7-4e94-a0a1-4e432cf3c977)
 Please see attachment for example. The clouds move in the picture too and during a storm there is lightning.

**Reply by papjul** [reply]
You want us to put your videos as background animation? Do you have one video for each weather condition we support?

Could you share them so I can check? Because unfortunately, I'm afraid they would make the app much much bigger, and I don't really see the added value; what's your use case? What would they do that the current weather animations don't already? Because vector animations are native and optimized compared to raster videos. Also, they feel more integrated, more app-like to me.

In either case, you could still make your own design through the Gadgetbridge WeatherSpec. Let us know if that would be something you're interested in and you need help with it.
